enginetest/queries.go

```diff
@@ -4114,6 +4114,10 @@ var QueryTests = []QueryTest{
 			{1, 1, 4},
 		},
 	},
+	{
+		Query:    `SELECT IS_UUID(UUID())`,
```

Comment:
For locality, I'd say you could move this to the `script_queries` test that you have for UUID, just so that they're all in the same place.